Makefile

```diff
@@ -61,7 +61,7 @@ requirements: ## install development environment requirements
 	pip-sync requirements/base.txt requirements/dev.txt requirements/private.* requirements/test.txt
 
 test: clean ## run tests in the current virtualenv
-	py.test
+	py.test tests/
```

Comment:
@jcdyer, this had previously been looping through all of the libraries installed in my virtualenv and `.tox/` as well as our tests - was that deliberate?